MonoGame.Framework/Storage/StorageContainer.cs

```diff
@@ -0,0 +1,475 @@
+// MonoGame - Copyright (C) MonoGame Foundation, Inc
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Xna.Framework.Storage
+{
+    //	User storage is usually in the "My Documents" folder of the user who is currently logged in, in the SavedGames folder.
+    //	A subfolder is created for each game according to the titleName passed to the OpenContainer method.
+    //	When no PlayerIndex is specified, content is saved in the AllPlayers folder. When a PlayerIndex is specified,
+    //	the content is saved in the Player1, Player2, Player3, or Player4 folder, depending on which PlayerIndex
+    //	was passed to BeginShowSelector.
+
+    /// <summary>
+    /// Contains a logical collection of files used for user-data storage.
+    /// </summary>			
+    /// <remarks>MSDN documentation contains related conceptual article: https://learn.microsoft.com/en-us/previous-versions/windows/xna/bb199074(v=xnagamestudio.40)</remarks>
+    public partial class StorageContainer : IDisposable
+    {
+        private readonly PlayerIndex? _playerIndex;
+
+        // In memory container
+        private Dictionary<string, byte[]> _containers;
```

Comment:
This IS the `StorageContainer`...  it contains files.

Should this not be a list of files in the container?  Not a list of containers.